.github/workflows/s4hana-build-connector-template.yml

```diff
@@ -0,0 +1,96 @@
+name: Build (S/4 HANA Connector)
+
+on:
+  workflow_call:
+    inputs:
+      hana-connector-group:
+        required: true
+        type: string
+
+jobs:
+  build-examples:
+    name: Build Examples
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v3
+
+      - name: Set Up Ballerina
+        uses: ballerina-platform/setup-ballerina@v1.1.0
+        with:
+          version: latest
```

Comment:
Don't we need to make this configurable?

Reply:
Fixed in. [326479b](https://github.com/ballerina-platform/ballerina-library/pull/6750/commits/326479b47d56fe0d098ac355827ae51801fb31a7)